src/commands/development/paginate.js

```diff
@@ -0,0 +1,92 @@
+import { Command } from 'discord.js-commando'
+import { RichEmbed } from 'discord.js'
+import { respondWithPaginatedEmbed } from '../../utils/embed'
+
+/*
+  Just some fun/silly made-up data to test pagination with.
+*/
+const items = [
+  { name: 'evan', value: 'so cool they even have their own emoji' },
+  { name: 'gusto', value: 'benevolent dictator for life <3' },
+  { name: 'elfayer', value: 'unrivaled at archery (except by Robin Hood)' },
+  { name: 'naito', value: 'has degree in TypeScript witchcraft' },
+  { name: 'lloyd', value: 'is practically neighbours with sustained' },
+  { name: 'akryum', value: 'is there anything he cannot do' },
+  { name: 'dobromir', value: '#most-attractive-moderator' },
+  { name: 'dr. taco', value: 'mexican food specialist and meme expert' },
+  { name: 'joe', value: 'has gifs for all possible occasions' },
+  { name: 'dave stewart', value: 'saviour of all vuex users' },
+  { name: 'galvez', value: 'best hat in all the lands' },
+]
+
+/*
+  This development-only command allows developers to test the pagination 
+  utility. It is only available on the development server.
+*/
+module.exports = class DevelopmentPaginateCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: 'paginate',
+      args: [
+        {
+          key: 'itemsPerPage',
+          type: 'integer',
+          prompt: 'how many items to display per page?',
+        },
+        {
+          key: 'observeReactionsFor',
+          type: 'integer',
+          prompt: 'how long (in ms) to allow pagination for?',
+        },
+        {
+          key: 'showDetailsInFooter',
+          type: 'boolean',
+          prompt: 'show current page etc. in the footer?',
+        },
+        {
+          key: 'inlineFields',
+          type: 'boolean',
+          prompt: 'make all fields inline?',
+        },
+        {
+          key: 'authorOnly',
+          type: 'boolean',
+          prompt: 'only allow the author to paginate?',
+        },
+      ],
+      examples: [
+        '!paginate 5 - Five items per page',
+        '!paginate 5 30000 - As above, plus: Pagination active for 30 seconds.',
```

Comment:
should we not just accept seconds? who would request something for less than a second

Reply:
I mean this command is for developers it's essentially a... unit test... for this command and I think JS developers are accustomed to working with milliseconds?

I presume at the user-level, if we ever wanted to allow them control over this parameter (I am not so sure we even would), then they'd pass in seconds and we'd multiply by 1000... so it goes.